core/domain/classifier_services.py

```diff
@@ -417,12 +433,35 @@ def _add_examples(self, examples, iterations):
     def add_examples_for_training(self, training_examples):
         """Adds examples to the classifier with _training_iterations number of
         iterations.
+
+        Args:
+            training_examples: list of examples. A single example is a doc
```

Comment:
Suggest putting quotes around examples (list of 'examples'), since this docstring is slightly unusual because 'examples' isn't a type.

Overall, suggested rewrite:
```
           training_examples: list of 'examples'. Each example is represented by a 2-element
                list. The first item of the list is a str representing a doc, and the second item is a
                list of labels that the doc should be matched to. E.g.:

                 ... example goes here ...
```